src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/email/EmailSendUseCase.java

```diff
@@ -1,9 +1,22 @@
 package de.dhbw.karlsruhe.students.mailflow.core.application.email;
 
-import de.dhbw.karlsruhe.students.mailflow.core.domain.email.Email;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.InvalidRecipients;
 import de.dhbw.karlsruhe.students.mailflow.core.domain.email.exceptions.MailboxLoadingException;
 import de.dhbw.karlsruhe.students.mailflow.core.domain.email.exceptions.MailboxSavingException;
 
 public interface EmailSendUseCase {
-    public void sendEmail(Email email) throws MailboxLoadingException, MailboxSavingException;
+
+  void sendPreparedEmail() throws MailboxLoadingException, MailboxSavingException;
```

Comment:
Should also throw something like `IllegalStateException` or the like if the caller hasn't constructed a valid e-mail

Reply:
I'm not sure how a unvalid e-mail would look like (e.g. for test cases) and how we can validate them? The only validation right now is that atleast one recipient is provided. 

It's a good idea to check it inside the sendPreparedEmail() again though!